prime-router/settings/STLTs/SD/sd-phd.yml

```diff
@@ -0,0 +1,92 @@
+- name: "sd-phd"
+  description: "South Dakota Public Health Department"
+  jurisdiction: "STATE"
+  stateCode: "SD"
+  countyName: null
+  filters: null
+  featureFlags: null
+  keys: null
+  senders: []
+  receivers:
+    - name: "full-elr"
+      organizationName: "sd-phd"
+      topic: "full-elr"
+      customerStatus: "active"
+      translation:
+        schemaName: "azure:/metadata/hl7_mapping/receivers/STLTs/SD/SD-receiver-transform.yml"
+        useTestProcessingMode: false
+        useBatchHeaders: false
+        receivingApplicationName: null
+        receivingApplicationOID: null
+        receivingFacilityName: null
+        receivingFacilityOID: null
+        messageProfileId: null
+        replaceValue: {}
+        replaceValueAwithB: {}
+        reportingFacilityName: null
+        reportingFacilityId: null
+        reportingFacilityIdType: null
+        suppressQstForAoe: false
+        suppressHl7Fields: null
+        suppressAoe: false
+        defaultAoeToUnknown: false
+        replaceUnicodeWithAscii: false
+        useBlankInsteadOfUnknown: null
+        truncateHDNamespaceIds: false
+        truncateHl7Fields: null
+        usePid14ForPatientEmail: false
+        convertTimestampToDateTime: null
+        cliaForOutOfStateTesting: null
+        cliaForSender: {}
+        phoneNumberFormatting: "STANDARD"
+        suppressNonNPI: false
+        processingModeCode: null
+        replaceDiiWithOid: null
+        applyOTCDefault: false
+        useOrderingFacilityName: "STANDARD"
+        valueSetOverrides: {}
+        nameFormat: "STANDARD"
+        receivingOrganization: null
+        convertPositiveDateTimeOffsetToNegative: false
+        stripInvalidCharsRegex: null
+        convertDateTimesToReceiverLocalTime: false
+        useHighPrecisionHeaderDateTimeFormat: false
+        type: "HL7"
+        truncationConfig:
+          truncateHDNamespaceIds: false
+          truncateHl7Fields: []
+          customLengthHl7Fields: {}
+      jurisdictionalFilter:
+        - "(Bundle.entry.resource.ofType(ServiceRequest)[0].requester.resolve().organization.resolve().address.state = 'SD') or (Bundle.entry.resource.ofType(Patient).address.state = 'SD')"
+      qualityFilter:
+        - "Bundle.entry.resource.ofType(Observation).value.coding.code != '260385009'"
```

Comment:
I think, we should add those common filters in here as well.

Reply:
easier way to write this is to use the abnormal flags as opposed to specific SNOMED codes since you could theoretically get results that have none of those SNOMEDs.

Example: Bundle.entry.resource.ofType(Observation).interpretation.coding.code = 'A'

I usually combine them in the condition filter with the condition code since there may be conditions where they want positives and negatives and others where they only want positives.

Condition code example:

If you are removing unwanted observations:

"%resource.ofType(Observation).where(interpretation.coding.code = 'A').code.coding.extension('https://reportstream.cdc.gov/fhir/StructureDefinition/condition-code').value.where(code in (''840539006' )).exists()"

If you are sending all observations as long as one qualifies:
"Bundle.entry.resource.ofType(Observation).where(interpretation.coding.code = 'A').code.coding.extension('https://reportstream.cdc.gov/fhir/StructureDefinition/condition-code').value.where(code in ('6142004' | '541131000124102' | '840539006' | '414015000' | '86406008')).exists()"

Reply:
@oslynn we don't add the common filters, those are the default filters and since they have their own filters, they are not using the default. At least that is how I understand it. 

@GilmoreA6 making the change, but for understanding's sake how are those three codes equivalent to A?